lib/benchee/benchmark/scenario.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Benchee.Benchmark.Scenario do
+  defstruct [:job_name, :function, :input_name, :input, :run_times,
+             :run_time_statistics, :memory_usages, :memory_usage_statistics]
+
+  @type t :: %__MODULE__{
+    job_name: binary | nil,
+    function: fun | nil,
```

Comment:
I believe whenever we create it it should have a fun and a name --> they should never be nil imo :)